<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>42</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result_value">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>165</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>179</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Сложение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sub">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Вычитание</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_mul">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Умножение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_div">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Деление</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_deg">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>255</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Степень</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sqrt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Корень</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>165</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_cos">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cos(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_tg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>255</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Tg(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_ctg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ctg(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_arcsin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>165</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Arcsin(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_arccos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Arccos(x)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>91</y>
      <width>389</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_op1_text">
         <property name="text">
          <string>Операнд 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_op1_value"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_op2_value">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_op2_text">
       <property name="text">
        <string>Операнд 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_do">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Калькулятор</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
